src/topics/resolver/ens-resolver.ts

```diff
@@ -20,30 +22,59 @@ export class EnsResolver extends GraphQLProvider implements IResolver {
       : (this.provider = ethers.getDefaultProvider());
   }
 
-  public async resolve(ensData: string): Promise<string> {
+  resolvedAccounts: FetchedData = {};
+
+  public async resolve(ensDataArray: FetchedData): Promise<FetchedData> {
+    const ensData = Object.keys(ensDataArray);
+
+    const resolveENSData = async (ensData: string[]): Promise<void> => {
+      const userData = await this.resolveLensHandlesQuery(ensData);
+
+      const resolvedAccounts = {} as FetchedData;
+
+      for (const user of userData) {
+        if (user.resolvedAddress !== null) {
+          resolvedAccounts[user.resolvedAddress.id] = ensDataArray[user.name];
+        } else {
+          const retryResolved = await this.resolveEnsFromJsonRpc(user.name);
+          if (retryResolved) {
```

Comment:
I think you need to test if `retryResolved` is different from "". I was able to have no errors with djsdjfdshbf.ens